Thanks, so my Instance is broken

Building frontend resources...
/tmp/build_244a3cdc/frontend /tmp/build_244a3cdc
> frontend@0.1.0 build /tmp/build_244a3cdc/frontend
> react-scripts build
Creating an optimized production build...
Failed to compile.
/tmp/build_244a3cdc/frontend/src/modules/search/withSearch.tsx
TypeScript error in /tmp/build_244a3cdc/frontend/src/modules/search/withSearch.tsx(38,13):
Type 'ReactNodeArray' is not assignable to type 'string | ReactNode[]'.
  The type 'readonly ReactNode[]' is 'readonly' and cannot be assigned to the mutable type 'ReactNode[]'.  TS2322
    36 |             );
    37 | 
  > 38 |             base = mark(base, rule, tag);
       |             ^
    39 |         }
    40 |     }
    41 |
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frontend@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the frontend@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /app/.npm/_logs/2021-10-15T21_36_18_822Z-debug.log
Collecting static files...
bin/post_compile: line 21: ./manage.py: No such file or directory
bin/post_compile: line 22: ./manage.py: No such file or directory
Current commit being deployed: 
fb46533dec5abf918e5f299cdff4306dbc93091c
bin/post_compile: line 29: static/revision.txt: No such file or directory

so automatic deploy thing - https://github.com/readingsnail/pontoon/compare/b5c8192d...fb46533d

--- a/bin/post_compile
+++ b/bin/post_compile
@@ -9,13 +9,14 @@ fi
 # Compile static assets
 export PATH=/app/.heroku/node/bin:$PATH
 npm install .
-pushd frontend && npm install && popd
+npm install -g yarn
+pushd frontend && yarn && popd
 
 echo "Running webpack..."
 ./node_modules/.bin/webpack -p
 
 echo "Building frontend resources..."
-pushd frontend && npm run build && popd
+pushd frontend && yarn run build && popd
 
 echo "Collecting static files..."
 ./manage.py migrate --noinput

